sdk/storage/azure-storage-blob-cryptography/src/main/java/com/azure/storage/blob/specialized/cryptography/EncryptedBlobClientBuilder.java

```diff
@@ -165,6 +167,9 @@ public EncryptedBlobAsyncClient buildEncryptedBlobAsyncClient() {
     }
 
     private HttpPipeline getHttpPipeline() {
+        CredentialValidator.validateSingleCredentialIsPresent(
```

Comment:
Can we also update the javadoc on `build*Client()` methods to document how this validation works?